laceworksdk/api/cloud_activities.py

```diff
@@ -29,49 +32,84 @@ def __init__(self, session):
     def get(self,
```

Comment:
Is the idea for paginated apis that `get()` will always return "all" and that if you actually want to work with pagination you would use a raw `get_pages()`?  Meaning we're not actually exposing iterables at the api layer?

Reply:
This is one of the things that I'm wrestling with on this update.  My dream was to give any existing integrations all pages of data for the price of "on the house," but due to the lack of properly utilizing a generator, I realize that there may be some significant performance and resource consumption issues related to that.  Would love some feedback on whether we should leave `get` alone and simply introduce new `get_pages` and `items`/`get_items` methods for pagination.

Reply:
I don't fully understand the backwards compatibility requirements as it pertains to whether or not we're adding paging to things we've already established that `get()` would return `all()`.

One approach could be that `get()` would be a generator and `all()` would return everything.  For things that aren't paginated we could just alias get->all.  

Otherwise you could establish `get()` as the thing that returns everything and for paginated resource introduce a `get_pages()` which would also seem to align with the library.